src/MPI.jl

```diff
@@ -57,6 +57,16 @@ else
     error("Unknown MPI binary: $(MPIPreferences.binary)")
 end
 
+if Sys.isunix()
```

Comment:
```suggestion
@static if Sys.isunix()
```
Dunno if this is even possible at module scope, but it was used in the `__init__` function

Reply:
Top-level expressions are evaluated at precompile time anyway, a top-level `@static` doesn't do anything special.

Reply:
OK, got it, thanks. Feel free to resolve this then ;-)